deps/patches/libssh2-mbedtls.patch

```diff
@@ -35,6 +35,47 @@ index 9d6c0be..8bc1292 100644
  
  install:
    - mkdir bin
+diff --git a/README b/README
+index 39abc20..8a14856 100644
+--- a/README
++++ b/README
+@@ -4,9 +4,9 @@ libssh2 - SSH2 library
+ libssh2 is a library implementing the SSH2 protocol, available under
+ the revised BSD license.
+ 
+-Web site: http://www.libssh2.org/
++Web site: https://www.libssh2.org/
```

Comment:
Yes hopefully it will, we haven't heard much there though right? If I had to guess, maybe the reason I wasn't able to get it to pass tests was because I was trying to use stock unmodified mbedtls instead of applying the config patch?